upgrading/automated_upgrades.adoc

```diff
@@ -368,9 +371,11 @@ warning that your hook needs to be updated to work with the latest
 - Hooks have no error handling, so an error in a hook halts the upgrade
 process. If you get an error, you must address the problem and then start the
 upgrade again.
+- You can run upgrade hooks on only nodes, not masters. To run the hooks on
+masters, you must specify a master hook for those nodes. 
```

Comment:
To be more precise, we could make it as   "You can run `node upgrade hooks` on only nodes, not masters. "